modules/about-administrator-perspective.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// web_console/web-console-overview.adoc
+
+:_content-type: CONCEPT
+[id="about-administrator-perspective_{context}"]
+= About the Administrator perspective in the web console
+
+The {product-title} web console provides two perspectives; the *Administrator* perspective and the *Developer* perspective. The *Administrator* perspective enables you to view the cluster inventory, capacity, general and specific utilization information, and the stream of important events, all of which help you to simplify planning and troubleshooting tasks. Both project administrators and cluster administrators can view the *Administrator* perspective.
```

Comment:
I'm not sure the first sentence is really necessary in this module, or in the module "About the Developer perspective".